SwiftWisdom/Core/StandardLibrary/Array/Array+Utilities.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 public extension Array {
-    public func ip_subArrayFromIndices(_ indices: [Int]) -> [Element] {
+    public func ip_subArrayFrom(indices: [Int]) -> [Element] {
```

Comment:
include from in the parameter `ip_subArray(fromIndices indices: [Int]) -> [Element]`